test/smoke-fails/atomic-contended/performBuildRun.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+usage() { echo "Usage: $0 [-a <AOMP directory>]" \
+                        " [-c <Compiler path>]" \
+                        " [-f <Compiler flags>]" \
+                        " [-t <Test source file>]" \
+          1>&2; exit 1; }
+
+while getopts ":a:c:f:t:" o; do
+    case "${o}" in
+        a)
+            AOMP=${OPTARG}
+            ;;
+        c)
+            CC=${OPTARG}
+            ;;
+        f)
+            CFLAGS=${OPTARG}
+            ;;
+        t)
+            TESTSRC=${OPTARG}
+            ;;
+        *)
+            usage
+            ;;
+    esac
+done
+
+# Set 'PROJECT_NAME' to the parent directory's name
+PROJECT_NAME=$(basename $(pwd))
```

Comment:
Should this do a `realpath` to work around symlinks as well?

Reply:
Good idea. Have to think how this might look like TBH.